testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1362,12 +1362,13 @@
   timeout_time = (Time.now + long_wait_delay + round_minute).strftime('%Y%m%d%H%M')
   node = get_system_name(client)
   $server.run("spacecmd -u admin -p admin clear_caches")
-  cmd = "spacecmd -u admin -p admin schedule_listcompleted #{current_time} #{timeout_time} #{node} | grep 'Package List Refresh scheduled by admin' | head -1"
+  cmd = "spacecmd -u admin -p admin schedule_list #{current_time} #{timeout_time} #{node} | grep 'Package List Refresh scheduled by admin'"
```

Comment:
```
schedule_list: List all actions
usage: schedule_list [BEGINDATE] [ENDDATE]
```
There is no "system" as parameter. So it makes no sense to give `#{node}` as parameter

Reply:
Thanks. Yes, this is good point. I am re-working this a bit, so it still will be possible to get status of package refresh specific for the selected system.